tf-modules/internal/cumulus-test-cleanup/cleanups3.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+
+const internalDeleteParams = [
+  { regex: /.*-\d{11,}-test-data\/.*/, olderThanDays: 2 },
```

Comment:
These seem ok, although I'm unsure how to evaluate anything that might have been deleted by mistake (as presumably it's been run?). 

I'm a little concerned about inadvertent cleanup or test situations that exceed these timeouts.... but perhaps the solution to that is to explicitly deploy to a bucket not covered by this name schema?

Reply:
It's fine to increase the time now that the original cleanup is done and we're not doing bucket versioning anymore. S3 is in a much more stable place at this point. When I ran it, I would have it print out what it's going to delete so I could do a manual check and did a lot of testing on it.

I could increase it to 14? 30? Hopefully the int tests are on a better path to being cleaned up.

Reply:
I think 2 weeks is probably well outside the norm for this sort of thing, so that seems fine 👍🏻